### PR TITLE
Timesync optimization fix

### PR DESCRIPTION
**Pull request by asahtik**
tsec param split in two to fix struct being bigger due to alignment

### CODE REVIEW

**Review by themarpe — APPROVED** [review]
LGTM otherwise, thanks!

File: include/XLink/XLinkPrivateDefines.h [hunk 1]
```diff
@@ -134,8 +134,9 @@ typedef struct xLinkEventHeader_t{
     eventId_t           id;
     xLinkEventType_t    type;
     char                streamName[MAX_STREAM_NAME_LENGTH];
-    uint64_t            tsec;
     uint32_t            tnsec;
+    uint32_t            tsecLsb;
+    uint32_t            tsecMsb;
```
Comment (szabi-luxonis) [inline include/XLink/XLinkPrivateDefines.h:139]: I wouldn't add new fields in the middle of a struct/enum. Much better to add them at the end.

Comment (asahtik) [reply include/XLink/XLinkPrivateDefines.h:139]: The timestamp parameters use memory that was previously occupied by the bigger stream name buffer (for backward compatibility with bootloader) so they have to be between streamName and streamId.

**Review by themarpe — APPROVED** [review]

**Review by szabi-luxonis — APPROVED** [review]

**Comment by themarpe** [comment]
Already merged in c940feaf9321f06a7d9660f28e686a9718135f38

**Pull request closed without merging**